core/spec/lib/spree/core/importer/order_spec.rb

```diff
@@ -424,7 +424,7 @@ module Core
 
       it 'adds line item adjustments from taxation' do
         line_items.first[:adjustments_attributes] = [
-          { label: 'Line Item Tax', amount: -4.99, tax: true }
+          { label: 'Line Item Tax', amount: 4.99, tax: true }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.